src/util/Bundles.js

```diff
@@ -196,18 +196,49 @@ define([
       return this.currentLanguage === language;
     },
 
+    /**
+     * Converts language code to a style of a key used in object which contains overrides e.g. 'nl-nl', 'nl-NL'
+     *
+     * @param overridesObject
+     * @param languageCode
+     * @returns {String} Converted language code
+     */
+
+    adjustLocalizationKey: function (overridesObject, languageCode) {
+      var adjustedLocalizationKey;
+
+      if(languageCode.split('-').length < 2) {
+        return languageCode;
+      }
+
+      adjustedLocalizationKey = languageCode.toLowerCase();
+
+      _.each(overridesObject, function (item, key) {
+        if(key.toLowerCase() !== key) {
+          adjustedLocalizationKey = _.map(languageCode.split('-'), function (keyPart, index) {
+            if(index === 1) {
+              return keyPart.toUpperCase();
+            }
+            return keyPart;
+          }).join('-');
+        }
+      });
+
+      return adjustedLocalizationKey;
+    },
+
     loadLanguage: function (language, overrides, assets) {
       var parsedOverrides = parseOverrides(overrides);
```

Comment:
The other solution is to modify `parseOverrides`. Always return the key in lowercase. The comparison will be case insensitive. Will let @samerfanek-okta to decide

Reply:
Yeah I think it makes more sense to put it in ``parseOverrides`` and return a lowercase key. Also, can you add unit tests for this covering the various combinations? (For example, 'nl', 'NL', 'nl-nl', 'nl-NL', etc..)